service/authentication/AuthenticationEvents.ts

```diff
@@ -7,7 +7,12 @@ const AuthenticationEvents = {
      * userIdentity - if user has been logged in then it contains user name. If
      *                contains 'null' or 'undefined' then user is not logged in.
      */
-    IDENTITY_UPDATED: 'authentication.identity_updated'
+    IDENTITY_UPDATED = 'authentication.identity_updated'
 };
 
-module.exports = AuthenticationEvents;
+export const IDENTITY_UPDATED = AuthenticationEvents.IDENTITY_UPDATED;
+
+// TODO: this was a pre-ES6 module using module.exports = AuthenticationEvents which doesn't translate well
```

Comment:
Ditto, let's make this change now, I'll kick off a full e2e test to be sure.

Reply:
Merging in 009 has created a fair bit of merge hell - I'll need to sort these out

Reply:
Given the issues with merging these I think it would be best to get them into typescript and then I'll do the updates to the consumers of the types. That way I won't have a number of branches trying to change the same files.

Will that be ok? I know we originally said we'd smash out the changes, but with lots of files needing to affect a number of places across the project the individual branch and PR approach isn't going to work well.